ISSD-3963

eliasantosf opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

kylembischof commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/en-US.html:
```diff
@@ -0,0 +1,72 @@
+
+<body>
+	<div class="header-logo" style="padding: 40px; text-align: center; margin: auto;"><img alt="Liferay Logo" class="logo"
+			src="http://localhost:18080/documents/d/PRM/liferay_logo_image-png" width="200" />
```
I believe `localhost:18080` should be replaced with `[$PORTAL_URL$]` similar to the other templates

eliasantosf replied:
I had already seen this problem, I ended up not making the substitutions after exporting, I'll send you the changes

eliasantosf replied:
Fixed

kylembischof commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/en-US.html:
```diff
@@ -0,0 +1,72 @@
+</head>
+
+<body>
+	<div class="header-logo" style="padding: 40px; text-align: center; margin: auto;"><img alt="Liferay Logo" class="logo"
```
sort div css styles

eliasantosf replied:
I thought for a moment that the gw formatSource indented the HTML lines of the templates. Fixed!

eliasantosf replied:
Fixed

kylembischof replied:
all css classes should be sorted, not just line 10

kylembischof commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/en-US.html:
```diff
@@ -0,0 +1,72 @@
+	<div class="header-logo" style="padding: 40px; text-align: center; margin: auto;"><img alt="Liferay Logo" class="logo"
+			src="http://localhost:18080/documents/d/PRM/liferay_logo_image-png" width="200" />
+	<div class="section-line" style="margin: auto; margin-top: 20px; margin-bottom: 20px; background-color: #6b6c7e; height: 1px; width: 592px;">
+			 </div>
```
fix indentations on divs and others

eliasantosf replied:
I thought for a moment that the gw formatSource indented the HTMl lines of the templates. Fixed!

eliasantosf replied:
Fixed

kylembischof commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/en-US.html:
```diff
@@ -0,0 +1,72 @@
+	<div class="main-container" style="margin: 0% 33%;">
+		<div>
+			<h1 class="text-primary" style="color: #377cff; font-size: 24px; margin-bottom: 16px;">Liferay MDF Request
+				Completed</h1>
```
do we normally do new lines like this? like `Liferay MDF Request` on one line but `Completed` on the next? it seems weird to cut it mid phrase. 

i think it should be one of these options:
```
<h1>
     text
</h1>
```
```
<h1>text</h1>
```

eliasantosf replied:
Fixed!

ryanschuhler commented:
@eliasantosf , can you make sure to run `blade gw formatSource` and `yarn format` on the site initializer to catch all the spacing/formatting issues?

eliasantosf commented:
@ryanschuhler, I ran `gw formatSouce` without the `blade`, I didn't have any suggestions for that. I ran it with the `blade` now and it suggested a few lines. I'll send you the changes.

ryanschuhler commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim.json:
```diff
@@ -7,6 +7,30 @@
 		"en_US": "MDF Claim"
 	},
 	"name": "MDFClaim",
+	"objectActions": [
```
@eliasantosf what does this action do? is it just a placeholder to add the logic later? also it may be helpful to make the externalReferenceCode something human readable, like the other notification action ercs

eliasantosf replied:
@ryanschuhler, as soon as the client extensions for the complete status of the request are in the master, I'll call it in the `objectActionExecutorKey`. I've already made the action definition ready for this, which is the case.

ryanschuhler replied:
great, makes sense, thanks!

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-26.liferay.com/job/test-portal-source-format/7715/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/7715/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9d4f105314736856fe8d7a915e1dcd9382438dcf">9d4f105314736856fe8d7a915e1dcd9382438dcf</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/eliasantosf/liferay-portal/tree/ISSD-3963">ISSD-3963</a><br/>Branch GIT ID: <a href="https://github.com/eliasantosf/liferay-portal/commit/84ddb05e8c1c3c50f77a1701f2d6bf1590035289">84ddb05e8c1c3c50f77a1701f2d6bf1590035289</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-26.liferay.com/job/test-portal-source-format/7715/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/7715/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-source-format/7715/">test-portal-source-format#7715</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-source-format/builds/7715/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-customer-delivery/liferay-portal/pull/58">liferay-customer-delivery#58</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2559538932">[master] ci:test:sf - eliasantosf &gt; liferay-customer-delivery - PR#58 - 2024-04-09[08:48:43]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2559538932<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/publish-testray-report/30416/">publish-testray-report#30416</a><br/></div></div>

eliasantosf commented:
@ryanschuhler, I removed the non-mandatory empty fields, leaving only the 
objectActionExecutorKey which is a mandatory field. And I changed the name of the erc

thaynam commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim.json:
```diff
@@ -7,6 +7,28 @@
 		"en_US": "MDF Claim"
 	},
 	"name": "MDFClaim",
+	"objectActions": [
```
you can add this one in a object-action directory
[see](https://github.com/liferay/liferay-portal/blob/cfe3d629d6384ff41b18b4bea57a4353bb2763ec/modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/object-actions)

eliasantosf replied:
Resolved

thaynam commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim.json:
```diff
@@ -7,6 +7,28 @@
+			},
+			"externalReferenceCode": "AUTOMATION-TO-COMPLETE-REQUEST-ACTION",
+			"label": {
+				"en_US": "Automation To Complete Request"
```
```suggestion
				"en_US": "MDF Claim Status Management"
```

thaynam replied:
we want to leave its name generic, so we can use the CE for other Status changes

eliasantosf replied:
Resolved

eliasantosf commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/mdf-claim.json:
```diff
@@ -7,6 +7,30 @@
+	"objectActions": [
+		{
+			"active": true,
+			"conditionExpression": "",
```
Do I send the Java code with the changes in this PR? Or would it be better in another PR?

eliasantosf replied:
Resolved

eliasantosf commented on modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/mdf-request-completed-template/notification-template.object-actions.json:
```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"active": true,
+		"conditionExpression": "mdfRequestStatus == \"completed\"",
```
Resolved

thaynam commented:
ci:forward:force

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites complete:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
AND If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest(master)/5701/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5701/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-33.liferay.com/job/test-portal-source-format/2896/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/2896/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/da0175e6eb9c61af012cedeaf57411a1c8933ebd">da0175e6eb9c61af012cedeaf57411a1c8933ebd</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/eliasantosf/liferay-portal/tree/ISSD-3963">ISSD-3963</a><br/>Branch GIT ID: <a href="https://github.com/eliasantosf/liferay-portal/commit/3109c30933ce7749884d6684da8dcfd45bf25b95">3109c30933ce7749884d6684da8dcfd45bf25b95</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-33.liferay.com/job/test-portal-source-format/2896/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/2896/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-33.liferay.com/job/test-portal-source-format/2896/">test-portal-source-format#2896</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-33.liferay.com/userContent/jobs/test-portal-source-format/builds/2896/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-customer-delivery/liferay-portal/pull/58">liferay-customer-delivery#58</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2560265054">[master] ci:test:sf - eliasantosf &gt; liferay-customer-delivery - PR#58 - 2024-04-11[12:43:35]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2560265054<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-33.liferay.com/job/publish-testray-report/12354/">publish-testray-report#12354</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 2 hours 8 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/da0175e6eb9c61af012cedeaf57411a1c8933ebd">da0175e6eb9c61af012cedeaf57411a1c8933ebd</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/da0175e6eb9c61af012cedeaf57411a1c8933ebd">da0175e6eb9c61af012cedeaf57411a1c8933ebd</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2560115516">EE Development Acceptance (master) - 488 - 2024-04-11[00:38:15]</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest(master)/5701/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5701/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest(master)/5701/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>A CI failure has occurred. QA Engineering has been notified. Please try again later or contact QA Engineering for an update.</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest(master)/5701/">test-portal-acceptance-pullrequest(master)#5701</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-36.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5701/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-customer-delivery/liferay-portal/pull/58">liferay-customer-delivery#58</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2560305824">[master] ci:test:relevant - eliasantosf &gt; liferay-customer-delivery - PR#58 - 2024-04-11[12:43:35]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2560305824<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-36.liferay.com/job/publish-testray-report/12465/">publish-testray-report#12465</a><br/></div></div>

thaynam commented:
ci:forward:force

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites complete:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
AND If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously completed test suites:
- `ci:test:relevant`
- `ci:test:sf`

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>stable</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/5636/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5636/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 34 out of 34 jobs passed in 1 hour 15 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/5402cbd24e14f1557c73fd0bc818368015045748">5402cbd24e14f1557c73fd0bc818368015045748</a></p><details><summary><strong>ci:test:stable - 34 out of 34 jobs PASSED</strong></summary><details><summary><strong>34 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/5636/">test-portal-acceptance-pullrequest(master)</a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/979745/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962452/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962448/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/358055/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1081585/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/855316/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962453/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/917248/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/4</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/979750/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/5</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/898121/">js-unit-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/979746/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962449/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/990380/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/979747/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/990382/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/917247/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/979749/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/855315/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/855314/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962451/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/855312/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962450/">modules-compile-jdk8/0/11</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/855313/">modules-compile-jdk8/0/12</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/572899/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1078799/">modules-integration-mysql57-jdk8_stable/0/1</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/1081584/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>playwright-compile-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962447/">playwright-compile-jdk8/0/0</a></li></ul></li><li>playwright-js-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/990381/">playwright-js-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/990383/">pmd-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/979748/">rest-builder-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/898120/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962446/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/962445/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></details></details><h5>For more details click <a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5636/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest(master)/5636/">test-portal-acceptance-pullrequest(master)#5636</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-35.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/5636/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>stable<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-customer-delivery/liferay-portal/pull/58">liferay-customer-delivery#58</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2560468549">[master] ci:test:stable - eliasantosf &gt; liferay-customer-delivery - PR#58 - 2024-04-12[09:21:42]</a><br/></div><div><strong>Testray Build ID:</strong><span></span>2560468549<br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-35.liferay.com/job/publish-testray-report/12455/">publish-testray-report#12455</a><br/></div></div>

thaynam commented:
ci:forward:force

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites complete:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
AND If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously completed test suites:
- `ci:test:relevant`
- `ci:test:sf`
AND Skipping previously passed test suites:
- `ci:test:stable`

liferay-continuous-integration commented:
All required test suite(s) completed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-39.liferay.com/job/forward-pullrequest/1245//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/149175
[Console](https://test-1-39.liferay.com/job/forward-pullrequest/1245//consoleFull)
